migrations/versions/0468_edit_limit_emails.py

```diff
@@ -0,0 +1,220 @@
+"""empty message
+
+Revision ID: 0468_edit_limit_emails
+Revises: 0467_add_limit_emails
+Create Date: 2016-06-01 14:17:01.963181
+
+"""
+
+from datetime import datetime
+
+from alembic import op
+from flask import current_app
+
+revision = "0468_add_limit_emails"
+down_revision = "0467_add_limit_emails"
+
+near_annual_limit_template_id = current_app.config["NEAR_ANNUAL_LIMIT_TEMPLATE_ID"]
+reached_annual_limit_template_id = current_app.config["REACHED_ANNUAL_LIMIT_TEMPLATE_ID"]
+annual_limit_updated_template_id = current_app.config["ANNUAL_LIMIT_UPDATED_TEMPLATE_ID"]
+quarterly_annual_limit_template_id = current_app.config["ANNUAL_LIMIT_QUARTERLY_USAGE_TEMPLATE_ID"]
+status_template_category_id = "55eb1137-6dc6-4094-9031-f61124a279dc"
+information_template_category_id = "207b293c-2ae5-48e8-836d-fcabd60b2153"
+template_ids = [
+    near_annual_limit_template_id,
+    reached_annual_limit_template_id,
+    annual_limit_updated_template_id,
+    quarterly_annual_limit_template_id,
+]
+
+annual_limit_reached = "\n".join(
+    [
+        "(la version française suit)",
+        "",
+        "Hello ((name)),",
+        "",
+        "0 ((message_type)) remaining until April  ((fiscal_end)).",
```

Comment:
Should we have `message_type_[en|fr]` so we can differentiate between: `email(s)`:`courriel(s)`?